colorfield/static/colorfield/colorfield.js

```diff
@@ -1,12 +1,16 @@
-/** global: django */
-
-window.addEventListener('load', function (event) {
-    if (typeof django !== 'undefined' && typeof django.jQuery !== 'undefined') {
-        (function ($) {
-            // add colopicker to inlines added dynamically
```

Comment:
Does `coloris` works correctly when adding inlines containing some color fields?

Reply:
I don't understand your question?

Reply:
See the comment in the previous code, when a model admin has inlines with `ColorField`, the color field widget should work correctly for both, widgets created on page load and also widgets created on the fly when adding a new inline.

Reply:
Yes, it's working fine